### PR TITLE
Work on the motivation episode

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: content/motivation.md [hunk 1]
```diff
@@ -6,60 +6,65 @@
...
 - Implements **branching**:
-  - you can work on several feature branches and switch between them
-  - different people can work on the same code/project without interfering
-  - you can experiment with an idea and discard it if it turns out to be a bad idea
+  - You can work on several feature branches and switch between them
+  - Different people can work on the same code/project without interfering
+  - You can experiment with an idea and discard it if it turns out to be a bad idea
 - Implements **merging**:
-  - tool to merge development branches for you
+  - Tool to merge development branches for you
```
Comment (rkdarst) [inline content/motivation.md:21]:
```suggestion
  - Person A and B's simultaneous work can be easily combined
```
suggestion: make less self-referential

File: content/motivation.md [hunk 2]
```diff
@@ -77,18 +82,18 @@ mylib-1.3.3_wk_13.8.08.tgz      somecode-20.7.09.tgz
 
 ### Collaboration
```
Comment (rkdarst) [inline content/motivation.md:84]:
```suggestion

With version control, none of these are needed anymore (or have much simpler answers):

```

File: content/motivation.md [hunk 3]
```diff
@@ -101,31 +106,22 @@ mylib-1.3.3_wk_13.8.08.tgz      somecode-20.7.09.tgz
...
+
+However, any version control is better than no version control and it is OK to prefer a different tool than Git.
+
+Other tools:
+- [Subversion](https://subversion.apache.org)
+- [Mercurial](https://www.mercurial-scm.org)
+
+Interesting newcomer:
+- [Pijul](https://pijul.org/)
 ```
```
Comment (rkdarst) [inline content/motivation.md:126]: What about acknowledging and discussing some difficulties?

```suggestion
## Difficulties of version control

Despite the benefits, let's be honest, there are some difficulties:

- One more thing to learn (It's probably worth it, will save you more time in the long run.  Basic career skill.)
- Difficult if some people don't want to use it (in the worst case, you can feed them versions)
- Advanced things can be difficult,a bit too many gotchas (basics are often enough, ask others for help when needed)
```

What other common difficulties are there we can acknowledge at the start??

**Review by rkdarst — APPROVED** [review]
Looks good, some ideas inspired by your starting point.

**Comment by bast** [comment]
Thanks for the additions! I will add more difficulties later.

**Pull request merged**